packages/manager/apps/okms/src/modules/secret-manager/routes/routes.tsx

```diff
@@ -73,80 +70,242 @@ const OrderOkmsModal = React.lazy(
 );
 
 export default (
-  <Route
-    path={SECRET_MANAGER_ROUTES_URIS.root}
-    Component={KmsLayout}
-    id={'secret-manager-root'}
-    errorElement={
-      <ErrorBoundary
-        redirectionApp="key-management-service"
-        isPreloaderHide={true}
-        isRouteShellSync={true}
-      />
-    }
-  >
+  <Route path={SECRET_MANAGER_ROUTES_URIS.root} id={'secret-manager-root'}>
     <Route index Component={Root} />
-    <Route path={SECRET_MANAGER_ROUTES_URIS.onboarding} Component={Onboarding} />
-    <Route path={SECRET_MANAGER_ROUTES_URIS.create} Component={CreateSecret}>
+    <Route
+      path={SECRET_MANAGER_ROUTES_URIS.onboarding}
+      Component={Onboarding}
+      handle={{
+        tracking: {
+          pageName: 'onboarding',
+          pageType: PageType.onboarding,
+        },
+      }}
+    />
```

Comment:
I am wondering if it would be worth creating a small utility function to do this, it would be cleaner here, and typesafe.

This:
```ts
      handle={{
        ...routeTracking('onboarding', PageType.onboarding),
      }}
    />
```


Or even this:

```ts
      handle={routeTracking('onboarding', PageType.onboarding)}
    />
```